packages/ragbits-core/src/ragbits/core/prompt/base.py

```diff
@@ -1,10 +1,10 @@
 from abc import ABCMeta, abstractmethod
-from typing import Dict, Generic, Optional, Type
+from typing import Any, Dict, Generic, Optional, Type
 
 from pydantic import BaseModel
 from typing_extensions import TypeVar
 
-ChatFormat = list[dict[str, str]]
+ChatFormat = list[dict[str, Any]]
```

Comment:
adding images to the prompt changes the format: if the images are present than content of message is a list of dictionaries, was trying to annotate it more precisely, however went through terrible things with mypy and done this change to speed things up. if precise annotations are important - let me know, however would prefer not to invest time into such things

Reply:
I think this is pretty important because this is a part of the public API of our library - we use `ChatFormat` in multiple places to document the required format for arguments / the return format of various methods.

Reply:
To be clear: I mean it as "it's work giving a good try", not "this is a must and we should spend an infinite amount of time on it" :)

Reply:
this has been resolved in the latest version